components/massmint/modals/MissingInfoModal.vue

```diff
@@ -14,7 +14,7 @@
           <div>{{ $t('massmint.required') }}</div>
 
           <div class="pl-3">
-            • {{ $t('massmint.incompleteNfts', { count: numMissingNames }) }}
+            • {{ $t('massmint.incompleteNfts', { count: missingNames.length }) }} ({{ missingNames.map(nft => nft.id).join(', ') }})
```

Comment:
It should looks like `#${nft.id}` according to https://github.com/kodadot/nft-gallery/issues/11320#issue-2765028112

Reply:
yeet https://github.com/kodadot/nft-gallery/pull/11340/commits/ed158ce7a25e662366fff75bd1aefa34bf8ee3a7